t/113-req-header-cookie.t

```diff
@@ -263,9 +263,10 @@ Cookie: boo=123; foo=bar
     }
 --- request
 GET /t
---- error_code: 500
---- error_log
-unsafe byte "0xa" in header value "boo=123\x0Afoo"
-failed to set header
+--- response_body
+Cookie foo: bar%0Dbar
+Cookie baz: 
+Cookie boo: 123%0Afoo
+Cookie: boo=123%0Afoo; foo=bar%0Dbar
 --- no_error_log
-[crit]
+[error]
```

Comment:
Please always add one and only one newline at the end of the test file (and other source files).